tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/TeamMailboxesContract.scala

```diff
@@ -2708,9 +2706,9 @@ trait TeamMailboxesContract {
       .setBody("testmail", StandardCharsets.UTF_8)
       .build
 
-    val response: Either[String, String] =
+    val response: Either[String, List[String]] =
```

Comment:
Just curious. What makes this contract change?

Reply:
Before: websocket receives only one message and stop
After: websocket receive 2 messages and stop

The real is that this test has 2 messages, but before the commit https://github.com/apache/james-project/pull/1921/files, the state of "sharesAccount" and "ownerAccount" is the same. But after it is different.